package-builder/extensions/mongodb/build.sh

```diff
@@ -9,6 +9,6 @@ echo "Building mongodb for gcp-php${SHORT_VERSION}"
 PNAME="gcp-php${SHORT_VERSION}-mongodb"
 
 # Download the source
-download_from_pecl mongodb
+download_from_pecl mongodb 1.9.1
```

Comment:
Why lock the version here as well?

Reply:
See my comment above about [apcu](https://github.com/GoogleCloudPlatform/php-docker/pull/533#discussion_r654014013)